ssas/service/admin/router.go

```diff
@@ -16,17 +18,40 @@ import (
 
 var infoMap map[string][]string
 var adminSigningKeyPath string
+var adminSigningKey string
 var server *service.Server
 
 func init() {
 	infoMap = make(map[string][]string)
 	adminSigningKeyPath = os.Getenv("SSAS_ADMIN_SIGNING_KEY_PATH")
+	adminSigningKey = os.Getenv("SSAS_ADMIN_SIGNING_KEY")
 }
 
 // Server creates an SSAS admin server
 func Server() *service.Server {
 	unsafeMode := os.Getenv("HTTP_ONLY") == "true"
-	server = service.NewServer("admin", ":3004", constants.Version, infoMap, routes(), unsafeMode, adminSigningKeyPath, 20*time.Minute, "")
+
+	var signingKey *rsa.PrivateKey
+
+	if adminSigningKey != "" {
```

Comment:
Should we add comments in the code, prior to the conditional, to explain whats going on?  
(The same comment applies in `public/router.go`)